File: src/main/java/cloud/api/RequestSensingData.java
package cloud.api;

import se.sics.kompics.address.Address;
import se.sics.kompics.network.Message;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-07-09
 *
 */

public class RequestSensingData extends Message {

	private static final long serialVersionUID = 8012003411245440962L;

	public RequestSensingData(Address source, Address destination) {
		super(source, destination);
	}

}

File: src/main/java/cloud/api/RequestSensingData.scala
package cloud.api

import se.sics.kompics.network.Message
import se.sics.kompics.address.Address

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class RequestSensingData(source:Address, destination:Address) extends Message(source, destination)